paper.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "PyMedPhys: A community effort to develop an open standard library for Medical Physics in Python"
+tags:
+  - Python
+  - Medical Physics
+  - Radiation Therapy
+  - Diagnostic Imaging
+  - DICOM
+authors:
+  - name: Simon Biggs
+    affiliation: 1
+  - name: Stuart Swerdloff
+    affiliation: 2
+  - name: Matthew Jennings
+    affiliation: 4
+  - name: Phillip Chlap
+    affiliation: "5, 6"
+  - name: Derek Lane
+    affiliation: 3
+  - name: Jacob Rembish
+    affiliation: 7
+  - name: Jacob McAloney
+    affiliation: 8
+  - name: Paul King
+    affiliation: 9
+  - name: Rafael Ayala
+    affiliation: 10
+  - name: Matthew Sobolewski
+    affiliation: "8, 11"
+affiliations:
+  - name: Radiotherapy AI, Australia
+    index: 1
+  - name: ELEKTA Pty Ltd, New Zealand
+    index: 2
+  - name: Elekta AB
+    index: 3
+  - name: Royal Adelaide Hospital, Australia
+    index: 4
+  - name: University of New South Wales, Australia
+    index: 5
+  - name: Ingham Institute, Australia
+    index: 6
+  - name: NYU Langone Health, USA
+    index: 7
+  - name: Riverina Cancer Care Centre, Australia
+    index: 8
+  - name: Painless Skin Cancer Treatment, Mississippi, USA
+    index: 9
+  - name: Hospital G.U. Gregorio Marañón, Spain
+    index: 10
+  - name: CancerCare Partners, Australia
+    index: 11
+date: 10 February 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+PyMedPhys is an open-source medical physics Python library built by a community
+that values and prioritizes code sharing, review, improvement, and peer
+development. The overall goal of PyMedPhys is to provide medical physicists
+around the world with a free, open-source library of Python applications to
+simplify and enhance both their research and clinical work. It is inspired by
+the collaborative work of our physics peers in astronomy and the Astropy
+Project [@astropy].
+
+# Statement of need
+
+As the treatment delivery methods used in radiation oncology continue to
+improve, the complexity associated with ensuring precise and accurate delivery
+continues to increase. In response, many medical physicists are turning toward
+software solutions to help maximize efficiency and quality of care. It is
+common for medical physicists to develop their own in-house code for performing
+clinical tasks; many of which have been standardized through recommendations by
+organizations such as the American Association of Physicists in Medicine
+(AAPM). This can often lead to the "reinvention of the wheel", where physicists
+are wasting time writing code to serve a purpose that someone else has already
+accomplished.
+
+By creating and promoting an open-source repository, PyMedPhys aims to improve
+the quality and accessibility of exisiting software solutions to problems faced by
+medical physicists. PyMedPhys is currently implemented around the world for a handful of
```

Comment:
Sometimes I wonder whether we should be marketing this to "medical physicists" or instead simply marketing PyMedPhys as a "medical physics toolkit". There is a subtle difference - the latter makes no implicit claim that the tasks traditionally considered within the scope of medical physics are/should only performed by medical physic**ists**.

Reply:
I would strongly support that language change! It feels more inclusive, though this might be a personal opinion since I am a physicist but not a Medical Physicist.